src/app/app-modules/nurse-doctor/visit-details/upload-files/upload-files.component.ts

```
@@ -309,17 +309,14 @@ export class UploadFilesComponent implements OnInit, DoCheck, OnChanges {
-          // window.open(url);
-          const a = document.createElement('a');
-          a.href = url;
-          a.download = result.fileName;
-          document.body.appendChild(a);
-          a.click();
-          document.body.removeChild(a);
+        const fileID = {
+          fileID: result,
+        };
+        this.labService.viewFileContent(fileID).subscribe((res: any) => {
+          if (res && res.data && res.data.statusCode === 200) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)